src/esn.inbox.libs/app/services/jmap-client-provider/jmap-client-provider.js

```diff
@@ -17,14 +16,14 @@ angular.module('esn.inbox.libs')
 
     function _initializeJmapClient() {
       return $q.all([
-        generateJwtToken(),
+        tokenAPI.getWebToken(true),
```

Comment:
```suggestion
        tokenAPI.getWebToken(),
```

we don't want a new token each time. we need to use the cached one